lib/worker.js

```diff
@@ -143,17 +167,32 @@ Worker.prototype.run = function () {
         merge(jobAndQueueData, { result: result }),
         'Task complete'
       )
+      if (process.env.WORKER_MONITOR) {
+        var tags = this._eventTags()
+        tags.result = 'success'
+        monitor.increment('ponos.finish', tags)
+      }
```

Comment:
@anandkumarpatel and I just had a little discussion. What do you think about doing something like:

```js
if (process.env.WORKER_MONITOR_DISABLED) {
  return
}
// do monitoring
```

Then it allows us to enable it via `HOST` and `PORT` environment variables, but still allows us to disable it if necessary (UDPocolypse or something)